solidity/contracts/token/HypERC721.sol

```diff
@@ -12,16 +12,25 @@ import {ERC721EnumerableUpgradeable} from "@openzeppelin/contracts-upgradeable/t
  * @author Abacus Works
  */
 contract HypERC721 is ERC721EnumerableUpgradeable, TokenRouter {
+    /**
+     * @param _mailbox Address of the mailbox contract that will process and handle remote transfers for this token.
+     */
     constructor(address _mailbox) TokenRouter(_mailbox) {}
 
     /**
      * @notice Initializes the Hyperlane router, ERC721 metadata, and mints initial supply to deployer.
+     *
+     * @dev The `_mintAmount` parameter is mostly used for a brand new NFT that want to exists only as a warp route.
+     * In other words, the entire warp route is deployed with HypLSP8, and no HypLSP8Collateral.
```

Comment:
would rather not have lukso-specific nomenclature
```suggestion
     * In other words, the entire warp route is deployed with HypERC721, and no HypERC721Collateral.
```

Reply:
Sorry my bad, I missed that